RInAppMessaging/Classes/Utilities/AtomicWrapper.swift

```diff
@@ -1,7 +1,8 @@
 /// This wrapper ensures synchronized access to the value only for getter and setter.
-/// Mutating functions, subscript, incrementation etc. are not synchronized.
+/// Mutating functions, subscript, incrementation etc. are not synchronized by default -
+/// use `mutate` function to ensure operation atomicity.
 @propertyWrapper
-struct AtomicGetSet<Value> {
+class AtomicGetSet<Value> {
```

Comment:
Do we intend to inherit from `AtomicGetSet`? If not, can it be `final class`?

Reply:
I'm setting it back to `struct`